src/bin/content.js

```diff
@@ -0,0 +1,226 @@
+
+const fs = require('fs')
+const path = require('path')
+const slugid = require('slugid')
+const parse= require('csv-parse/lib/sync')
+
+const coursesFile = process.argv[2]
+const modulesFile = process.argv[3]
+const eventsFile = process.argv[4]
+
+const ALL_GRADES = [
+    "AA",
+    "AO",
+    "EO",
+    "HEO",
+    "SEO",
+    "G6",
+    "G7",
+    "SCS"
+];
+
+function parseCourses(file) {
+    const rawData = fs.readFileSync(file);
+    const lines = parse(rawData.toString());
```

Comment:
The canonical way of doing this is to specify `{encoding: 'utf8'}` as a parameter to the `fs.readFileSync()` call instead of having to call `.toString()` explicitly on the `Buffer` object.